lib/mongodb.js

```diff
@@ -363,6 +363,26 @@ MongoDB.prototype.coerceId = function(model, id) {
 };
 
 /**
+ * Convert all objectIds
```

Comment:
Here you can say something like `coerce type of all object id properties to mongoDB's ObjectID`